framework/util/create_mml.pl

```diff
@@ -83,6 +84,7 @@ =head1 OPTIONS
 
 pod2usage(1) unless defined $cmd_opts{p} and defined $cmd_opts{c} and defined $cmd_opts{o} and defined $cmd_opts{b};
 
+# TODO: Unused parameter: PID
```

Comment:
I kept it for backward-compatibility. Otherwise, it is safe to remove the parameter -- and we should do so for the next major release.